pipes/WDL/workflows/sarscov2_genbank_ingest.wdl

```diff
@@ -0,0 +1,44 @@
+version 1.0
+
+workflow genbank_dump {
+    input {
+        File  Google_Maps_API_Key_File
+        String  user_email
+    }
+    call pull_data {
+        input:
+            Google_Maps_API_Key_File = Google_Maps_API_Key_File,
+            user_email = user_email
+    }
+    output {
+        File    seqs_fasta = pull_data.genbank_seqs_fasta
+        File    seqs_metadata = pull_data.genbank_seqs_metadata
+    }
+}
+
+
+task pull_data {
+
+    input {
+        File  Google_Maps_API_Key_File
+        String  user_email
+    }
+
+    command {
+        python3 ~/scripts/genbank_dump.py -k ~{Google_Maps_API_Key_File} -e ~{user_email}
+    }
+
+    output {
+      File genbank_seqs_fasta    = 'genbank_seqs.fasta'
```

Comment:
May be nice to include the date in the outputs so we know when they're from (ex. rename in the `command` block via something like `genbank_seqs_$(date --iso-8601=date).fasta` & `genbank_seq_metadata_$(date --iso-8601=date).tsv`), but that can happen in a future PR.